runtime/compiler/env/VMJ9.cpp

```diff
@@ -7729,6 +7729,19 @@ TR_J9VM::inlineNativeCall(TR::Compilation * comp, TR::TreeTop * callNodeTreeTop,
             if (!ReferenceClass)
                return 0;
 
+            int32_t offset =
+               getInstanceFieldOffset(ReferenceClass, REFERENCEFIELD, REFERENCEFIELDLEN, REFERENCERETURNTYPE, REFERENCERETURNTYPELEN, J9_RESOLVE_FLAG_INIT_CLASS);
+
+            // Guard against the possibility that the relevant field no longer exists
```

Comment:
Under what circumstance could the field no longer exist?

Reply:
I will update the comment to make that more clear.

Reply:
Ah I see, it's not that it could change during runtime; I was more concerned from a relocation/validation point of view.

Reply:
Fixed in commit b03ac952857a54bfc3169e631a9d866990949f81